pubspec.yaml

```diff
@@ -1,17 +1,18 @@
 name: storybook_flutter
 description: A storybook for Flutter widgets. Live preview of isolated widgets for faster development and showcase.
-version: 0.1.5
+version: 0.2.0
 homepage: https://github.com/ookami-kb/storybook_flutter
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0-133.2.beta <3.0.0'
 
 dependencies:
-  provider: ">=4.0.0 <5.0.0"
-  recase: ">=2.0.1 <4.0.0"
-  rxdart: ">=0.24.0 <1.0.0"
+  collection: ^1.15.0-nullsafety.5
```

Comment:
`collection` is a well tested and stable package, I think it's better than writing the code for `groupBy` yourself. Also the migration tool suggested to use it.